### ISSUE TITLE
When editing a Question there is no way to cancel the edit

### ISSUE THREAD

**Issue by jdisser** [haxwell/quizki#227]
There is no way to abort an edit once you reach the Edit Question page. A cancel button would be expected.

**Comment by haxwell** [comment]
I agree that would be nice.. for now, you can just click on the Quizki logo at the top right.
